site/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1 @@
+CODE_OF_CONDUCT.md
```

Comment:
This is a broken symlink. What was the intent here?

Reply:
This can be removed... I'm not sure how to remove it honestly